plugins/meta/flannel/flannel_windows_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2015 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"testing"
+	"net"
+	"github.com/stretchr/testify/assert"
```

Comment:
Almost all of CNI, including the other tests of the `flannel` package, are using Ginkgo.  Is there a reason that won't work here?

Reply:
Moved to tests to ginkgo.